openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx

```diff
@@ -567,10 +567,10 @@ const DashboardDetails = ({
                       />
                     </div>
                   </div>
-                  <div className="tw-table-responsive tw-my-6">
+                  <div className="tw-table-responsive tw-my-6 tw-bg-white tw-border tw-border-main tw-rounded tw-shadow">
```

Comment:
```css
tw-table-responsive tw-my-6 tw-bg-white tw-border tw-border-main tw-rounded tw-shadow
```
As we are using these classes in all places so can we create one common class and use it everywhere?

Reply:
Made requested changes